cmd/template/dbdriver/files/docker-compose/sqlite.tmpl

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  sqlite:
+    image: ujstor/sqlite-test
```

Comment:
This is a bit of a smell.
I mentioned in an earlier comment it might be better to just have a dedicated Docker file for this

Reply:
I am wondering if there's a need for SQLite. If you curl localhost:8080/health and there is no error, and if the DB_URL env is set (for example, to ./test.db), a curl command would create db in the root directory